src/sentry/integrations/jira_server/client.py

```diff
@@ -36,8 +36,8 @@ class JiraServerClient(IntegrationProxyClient):
     ISSUE_TYPES_URL = "/rest/api/2/issue/createmeta/%s/issuetypes"
     # https://docs.atlassian.com/software/jira/docs/api/REST/9.11.0/#api/2/priority-getPriorities
     PRIORITIES_URL = "/rest/api/2/priority"
-    # https://docs.atlassian.com/software/jira/docs/api/REST/9.11.0/#api/2/project/{projectKeyOrId}/priorityscheme
-    PRIORITY_SCHEME_URL = "/rest/api/2/project/%s/priorityscheme"
+    # https://docs.atlassian.com/software/jira/docs/api/REST/9.11.0/#api/2/priorityschemes-getPrioritySchemes
+    PRIORITY_SCHEMES_URL = "/rest/api/2/priorityschemes?expand=schemes.projectKeys&maxResults=200"
```

Comment:
The default maximum results is 100, but I changed it to 200 to be safe. I'd be very surprised if anybody has more priority schemes than that